src/modules/get-todos.js

```diff
@@ -0,0 +1,40 @@
+const todos = [
+  {
+    description: 'Complete Todo Project',
+    completed: false,
+    index: 0,
+  },
+  {
+    description: 'Read WebPack Tutorial',
+    completed: true,
+    index: 1,
+  },
+  {
+    description: 'Do Laundry',
+    completed: true,
+    index: 2,
+  },
+];
```

Comment:
- [ ] **Required:** You have rendered the list excellently to the DOM, however, you are required to render them in the order of their index.

**_Hint:_** Consider `Array.sort((a, b) => a.index - b.index`